velox/expression/tests/SignatureBinderTest.cpp

```diff
@@ -30,6 +30,26 @@ void testSignatureBinder(
   ASSERT_TRUE(expectedReturnType->kindEquals(returnType));
 }
 
+TEST(SignatureBinderTest, decimals) {
+  // decimal(10, 3), decimal(10, 5) -> decimal()
+
+  auto signature =
+      exec::FunctionSignatureBuilder()
+          .returnType("SHORT_DECIMAL(r_precision, r_scale)")
```

Comment:
Would it make sense to allow case-insensitive type names, e.g. short_decimal?

Reply:
The implementation should allow both. I will add tests to confirm.

Reply:
done